### PR TITLE
feat: added swaycwd script and using it in foot.sh

### PR DESCRIPTION
**Pull request by fraunos**
https://github.com/Manjaro-Sway/manjaro-sway/issues/198
I thought about adding it as a dependency from AUR, but I had no idea how to do it, so I just copied it over.

### CODE REVIEW

**Comment by fraunos** [comment]
I'm not sure about the dependencies, it needs `jq` for example - how do I make sure it is available?

**Comment by boredland** [comment]
ya, we should put that into a package instead. will do.

File: community/sway/usr/share/sway/scripts/foot.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ if [ -f $USER_CONFIG_PATH ]; then
     USER_CONFIG=$USER_CONFIG_PATH
 fi
 
-foot -c ${USER_CONFIG:-"/usr/share/sway/templates/foot.ini"} $@ &
+foot -D $(/usr/share/sway/scripts/swaycwd.sh) -c ${USER_CONFIG:-"/usr/share/sway/templates/foot.ini"} $@ &
```
Comment (boredland) [inline community/sway/usr/share/sway/scripts/foot.sh]:
```suggestion
foot -D $(swaycwd) -c ${USER_CONFIG:-"/usr/share/sway/templates/foot.ini"} $@ &
```

File: community/sway/usr/share/sway/scripts/swaycwd.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```
Comment (boredland) [inline community/sway/usr/share/sway/scripts/swaycwd.sh]: pls remove then

**Review by boredland — APPROVED** [review]

**Comment by boredland** [comment]
@fraunos I did go another way than you, hope you like it anyway. Installing swaycmd remains optional, it just falls back to $HOME. The old behavior is available on `$mod+Shift+Return`.

**Comment by boredland** [comment]
@all-contributors please add @fraunos for writing code to support swaycwd

**Comment by fraunos** [comment]
@boredland 
Sure, thanks! I hope others find this feature helpful.
Btw I just copied someones' code from AUR (I'm not sure what was the license, I included the source in a comment), I'm not sure this deserves a contributor status, but anyway, maybe I'll deserve it some day.
Love Your work, this edition is awesome, thanks!

**Pull request merged**